src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/JJDisputeService.java

```diff
@@ -295,7 +295,7 @@ public JJDispute setStatus(String id, JJDisputeStatus jjDisputeStatus, Principal
 		jjDisputeRepository.setStatus(jjDisputeToUpdate.getTicketNumber(), jjDisputeStatus, principal.getName());
 
 		// Set remarks with user's full name if a remark note is provided along with the status update
-		if(!StringUtils.isBlank(remark)) {
+		if(remark != null && !StringUtils.isBlank(remark)) {
```

Comment:
:) yeah, so used to Java where StringUtils.isBlank is a null-safe method.

Reply:
I think you're right. I thought this is Java 11's isBlank() method and it is not null safe based on this: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank()

However, what we used in the code is actually Apache Commons' StringUtils.isBlank() method so that's null safe. I can remove that extra null check.